scripts/us_census/acs5yr/subject_tables/s0101/S0101_spec.json

```diff
@@ -0,0 +1,287 @@
+      "measuredProperty": "age",
+      "statType": "medianValue",
+      "unit": "Years"
+    }
+  },
+  "pvs": {
+    "statType": {
+      "Margin of Error": "marginOfError",
+      "Percent MOE": "marginOfError"
+    },
+    "Node": {
+      "Child dependency ratio": "dcid:Count_Person_17OrLessYears_AsAFractionOf_Count_Person_18To64Years",
+      "Old-age dependency ratio": "dcid:Count_Person_65OrMoreYears_AsAFractionOf_Count_Person_18To64Years",
+      "Sex ratio (males per 100 females)": "dcid:Count_Person_Male_AsAFractionOf_Count_Person_Female"
+    },
+    "measuredProperty": {
+      "Child dependency ratio": "count",
+      "Old-age dependency ratio": "count",
+      "Sex ratio (males per 100 females)": "count"
+    },
+    "age": {
+      "10 to 14 years": "[10 14 Years]",
+      "15 to 17 years": "[15 17 Years]",
+      "15 to 19 years": "[15 19 Years]",
+      "15 to 44 years": "[15 44 Years]",
+      "16 years and over": "[16 - Years]",
+      "18 to 24 years": "[18 24 Years]",
+      "18 years and over": "[18 - Years]",
+      "20 to 24 years": "[20 24 Years]",
+      "21 years and over": "[21 - Years]",
+      "25 to 29 years": "[25 29 Years]",
+      "30 to 34 years": "[30 34 Years]",
+      "35 to 39 years": "[35 39 Years]",
+      "40 to 44 years": "[40 44 Years]",
+      "45 to 49 years": "[45 49 Years]",
+      "5 to 14 years": "[5 14 Years]",
+      "5 to 9 years": "[5 9 Years]",
+      "50 to 54 years": "[50 54 Years]",
+      "55 to 59 years": "[55 59 Years]",
+      "60 to 64 years": "[60 64 Years]",
+      "62 years and over": "[62 - Years]",
+      "65 to 69 years": "[65 69 Years]",
+      "65 years and over": "[65 - Years]",
+      "70 to 74 years": "[70 74 Years]",
+      "75 to 79 years": "[75 79 Years]",
+      "75 years and over": "[75 - Years]",
+      "80 to 84 years": "[80 84 Years]",
+      "80 to 84 years": "[80 84 Years]",
+      "85 years and over": "[85 - Years]",
+      "Under 18 years": "[- 18 Years]",
+      "Under 5 years": "[- 5 Years]",
+      "Child dependency ratio": "[- 17 Years]",
+      "Old-age dependency ratio": "[65 - Years]",
+      "60 years and over" : "[60 - Years]",
+      "62 years and over" : "[62 - Years]"
+    },
+    "gender": {
+      "Male": "Male",
+      "Male MOE": "Male",
+      "Female": "Female",
+      "Female MOE": "Female",
+      "Sex ratio (males per 100 females)": "Male"
+    },
+    "measurementDenominator": {
+      "Age dependency ratio": "Count_Person_18To64Years",
+      "Child dependency ratio": "Count_Person_18To64Years",
+      "Old-age dependency ratio": "Count_Person_18To64Years",
+      "Sex ratio (males per 100 females)": "Count_Person_Female"
+    },
+    "scalingFactor": {
+      "Age dependency ratio": "100",
+      "Child dependency ratio": "100",
+      "Old-age dependency ratio": "100",
+      "Sex ratio (males per 100 females)": "100"
+    }
+  },
+  "enumSpecializations": {
+  },
+  "inferredSpec": {},
+  "universePVs": [],
```

Comment:
Since `enumSpecializations`, `inferredSpec` and `universePVs` are empty, perhaps we can remove them?
I guess, the code to generate the column map handles the case where certain keys are missing by initialising them to empty dictionary (or list).